src/app/components/MediaLoader/configs/clipMedia.ts

```diff
@@ -88,12 +92,21 @@ export default ({
   };
 
   const items = [{ versionID, kind, duration: rawDuration }];
+
   if (showAds) items.unshift({ kind: 'advert' } as PlaylistItem);
 
+  const embedUrl = getEmbedURL({
+    mediaId: `${id}/${versionID}/${lang}`,
+    type: 'avEmbed',
+    isAmp,
```

Comment:
Does isAmp need to be passed here, or can we use context within the function to grab it?

Reply:
Ahh yeah this is a function nvm